lib/Screens/profile.dart

```diff
@@ -10,13 +13,365 @@ class Profile extends StatefulWidget {
+                                Text(profileData["weight"].toString()),
+                                Text(profileData["age"].toString()),
+                                Text(profileData["height"]),
+                                Text(profileData["dateJoined"]),
+                              ],
+                            ),
+                          ],
+                        ),
+                      )
+                    ],
+                  ),
+                ),
+              ),
+              const SizedBox(height: 10),
+              Card(
+                elevation: 10,
+                shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(16)),
+                child: Padding(
+                  padding: const EdgeInsets.all(5),
+                  child: Column(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      const Text(
+                        "Fitness Info",
+                        style: TextStyle(fontSize: 26),
+                      ),
+                      const SizedBox(height: 5),
+                      Padding(
+                        padding: const EdgeInsets.only(
+                          left: 20,
+                          right: 20,
+                          bottom: 10,
+                        ),
+                        child: Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                          children: [
+                            Column(
+                              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                              children: const [
+                                Text("Fitness Style:"),
+                                Text("Total Calories Burned:"),
+                                Text("Desired Weight:"),
+                                Text("Challenges Completed:")
+                              ],
+                            ),
+                            Column(
+                              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                              children: [
+                                Text(profileData["fitnessStyle"]),
+                                Text(profileData["caloriesBurned"].toString()),
+                                Text(profileData["desiredWeight"].toString()),
+                                Text(profileData["challengesCompleted"]
+                                    .toString()),
+                              ],
+                            ),
+                          ],
+                        ),
+                      )
+                    ],
+                  ),
+                ),
+              ),
+            ],
+          ),
+        ),
+        Positioned(
+          bottom: 10,
+          right: 10,
+          child: FloatingActionButton(
+            backgroundColor: tfstyle.trifitColor[700],
```

Comment:
You can actually remove "as tfstyle" from the import and then you don't need to reference it to access the styles. I thought you had to do that initially but you don't